spec/inquisition/plugin_spec.rb

```diff
@@ -0,0 +1,33 @@
+RSpec.describe Inquisition::Plugin do
+  subject(:dummy) do
+    Class.new do
+      include Inquisition::Plugin
+    end
+  end
+
+  describe '.collection' do
+    context 'when there are no descendants' do
+      it { expect(dummy.collection).to be_empty }
+    end
+
+    context 'when there is at least one descendant' do
+      let(:descendant) do
+        Class.new(dummy)
+      end
+
+      it do
```

Comment:
it's hard to read this example, i would make it
```expect { descendant }.to change(dummy.collection, :size).by(1)```

Reply:
Fixed https://github.com/rubygarage/inquisition/pull/74/commits/8832c1308cc49c1976bfe623eb0d3aea110ce53c#diff-475fb638797e13b4dcc5149eeb7cb8cdR18